data/35e/wizards_of_the_coast/supplement/tome_of_magic/tm_abilities.lst

```diff
@@ -1,12 +1,12 @@
-# CVS $Revision: $ $Author: $ -- Sun Aug 17 15:41:15 2014 -- reformated by prettylst.pl v1.51 (build 24906)
+# CVS $Revision: $ $Author: $ -- Sun Aug 17 15:41:15 2014 -- reformated by prettylst.pl v1.51 (build 24906)SOURCEPAGE:p.10SOURCELONG:Tome of Magic
 
 ###Block: Binder - Submitted by Dave
 # Ability Name			Unique Key						Category of Ability		Type												Visible	Define						Description																																																																																														Ability											Bonus Ability Pool								Source Page
-Weapon and Armor Proficiency	KEY:Binder ~ Weapon and Armor Proficiency	CATEGORY:Special Ability	TYPE:Class Feature.Binder Class Feature.SpecialQuality												DESC:As a binder, you gain proficiency with all simple weapons and with light armor, but not with shields.																																																																														ABILITY:Internal|AUTOMATIC|TYPE=ArmorProfLight|TYPE=WeaponProfSimple												SOURCEPAGE:p.10
+Weapon and Armor Proficiency	KEY:Binder ~ Weapon and Armor Proficiency	CATEGORY:Special Ability	TYPE:Class Feature.Binder Class Feature.SpecialQuality												DESC:As a binder, you gain proficiency with all simple weapons and with light armor, but not with shields.																																																																														ABILITY:Internal|AUTOMATIC|TYPE=ArmorProfLight|TYPE=WeaponProfSimple												SOURCESHORT:TM
```

Comment:
Why are we removing the page number?

Reply:
that was a mistake, and linked to the previous SOURCE tags mistake - I use 'vim' to edit and switch all files over to multi line editing for ease, and managed to move one and join two by mistake whilst editing this one.
As multi-line format can be enabled via a PCC option, there is no compatibility issue in using this, however by default I will revert back to single line format when saving - I assume that that is still your preference?
again, recommited to fix